apps/web-giddh/src/app/contact/utility/contact.store.ts

```
@@ -0,0 +1,64 @@
+
+import { Injectable, OnDestroy } from "@angular/core";
+import { ComponentStore, tapResponse } from "@ngrx/component-store";
+import { Observable, switchMap, catchError, EMPTY } from "rxjs";
+import { ToasterService } from "../../services/toaster.service";
+import { BaseResponse } from "../../models/api-models/BaseResponse";
+import { ContactService } from "../../services/contact.service";
+
+export interface ContactState {
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)